.travis.yml

```diff
@@ -0,0 +1,16 @@
+addons:
+  apt:
+    packages:
+      - xvfb
+
+before_install:
+  ## Install latest version of node
+  - nvm install node
```

Comment:
We shouldn't need this if the language is specified, look at the main repo's travis config. We can actually just specify multiple node versions